src/server/game/Handlers/TaxiHandler.cpp

```diff
@@ -107,8 +107,7 @@ void WorldSession::SendTaxiMenu(Creature* unit)
     GetPlayer()->SetTaxiCheater(lastTaxiCheaterState);
 }
 
-void WorldSession::SendDoFlight(uint32 mountDisplayId, uint32 path, uint32 pathNode)
-{
+void WorldSession::SendDoFlight(uint32 mountDisplayId, uint32 path, uint32 pathNode) {
```

Comment:
?

Reply:
I don't remember changing that line. 😂
Must be the IDE but either way I've changed it back to your preference.